sidecar.go

```diff
@@ -61,6 +61,7 @@ const (
 	TelegrafSecretAnnotationValue = "telegraf-operator"
 	TelegrafSecretDataKey         = "telegraf.conf"
 	TelegrafSecretLabelClassName  = TelegrafClass
+	TelegrafSecretLabelPodName    = "telegraf.influxdata.com/pod-name"
```

Comment:
What about `telegraf.influxdata.com/pod` instead of adding `-name`?

Reply:
Good idea. I like it more. Updated.